src/components/RadioButtons.tsx

```diff
@@ -1,36 +1,34 @@
 import React, {useState} from 'react';
 import {View} from 'react-native';
-import PropTypes from 'prop-types';
-import _ from 'underscore';
 import RadioButtonWithLabel from './RadioButtonWithLabel';
 import styles from '../styles/styles';
 
-const propTypes = {
+type Choice = {
+    label: string;
+    value: string;
+};
+
+type RadioButtonsProps = {
     /** List of choices to display via radio buttons */
-    items: PropTypes.arrayOf(
-        PropTypes.shape({
-            label: PropTypes.string.isRequired,
-            value: PropTypes.string.isRequired,
-        }),
-    ).isRequired,
+    items: Choice[];
 
     /** Callback to fire when selecting a radio button */
-    onPress: PropTypes.func.isRequired,
+    onPress: (value: string) => void;
 };
 
-function RadioButtons(props) {
+function RadioButtons({items, onPress}: RadioButtonsProps) {
     const [checkedValue, setCheckedValue] = useState('');
 
     return (
         <View>
-            {_.map(props.items, (item, index) => (
+            {items.map((item) => (
                 <RadioButtonWithLabel
-                    key={`${item.label}-${index}`}
+                    key={item.value}
```

Comment:
Are values always unique?

Reply:
@blazejkustra Since it's radio buttons where only one option can be selected, and selecting logic is based on the `item.value` [`isChecked={item.value === checkedValue}`](https://github.com/Expensify/App/pull/29970/files#diff-05b1a923fc4c79c5012fd8cab65b59ab905d30b3e0986eb792971ca792268198R27), I assume so. Otherwise, there will be two same values in radio buttons, which will cause problems anyway.